### DISCUSSION TITLE
How to add plugin in a docker setup?

### DISCUSSION THREAD

**Discussion by arladmin** [YOURLS/YOURLS#3240] [Q&A] [answered]
I am using the default docker image for my setup.

I would like to add extra plugins now.
e.g. this one: https://github.com/YOURLS/404-if-not-found

In the setup, it says to copy the files to a folder in the ```/user/plugins/``` folder.
Where's this folder in a docker context?

**Comment by LeoColomb** [comment, marked as answer]
You can create and bind a Docker volume for `/usr/src/yourls/user/plugins`.
